test/Reflection/FileReflectionTest.php

```diff
@@ -18,14 +18,6 @@
  */
 class FileReflectionTest extends \PHPUnit_Framework_TestCase
 {
-    public function testFileConstructor()
-    {
-        $filePath = __DIR__ . '/../../vendor/zendframework/zend-version/src/Version.php';
-        require_once $filePath;
-        $reflectionFile = new FileReflection($filePath);
-        $this->assertEquals(get_class($reflectionFile), 'Zend\Code\Reflection\FileReflection');
-    }
```

Comment:
yeah. Testing that ``new ...`` gives you an object of this class is either testing PHP (useless) or testing that there is no exception when passing a file (already covered by other tests as they would break)